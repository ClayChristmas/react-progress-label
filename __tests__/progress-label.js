import React from 'react';
import renderer from 'react-test-renderer';
import Nested from '../__fixtures__/nested';

test('nested', () => {
  expect(renderer.create(<Nested />).toJSON()).toMatchSnapshot();
});
