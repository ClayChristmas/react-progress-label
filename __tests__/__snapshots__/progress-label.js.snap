// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested 1`] = `
<div
  className="m-nested"
>
  <div
    className="example"
  >
    <svg
      className="label-1"
      height={250}
      viewBox="0 0 250 250"
      width={250}
    >
      <circle
        cx={125}
        cy={125}
        r={113}
        style={
          Object {
            "fill": "#333333",
            "stroke": "#330000",
            "strokeWidth": 24,
          }
        }
      />
      <path
        d="M 125 12 A 113 113, 0, 0, 1, 191.41973350904948,216.41892036436906"
        style={
          Object {
            "fill": "none",
            "stroke": "#ff0000",
            "strokeWidth": 24,
          }
        }
      />
      <circle
        cx={125}
        cy={12}
        fill="#ff0000"
        r={12}
      />
      <circle
        cx={191.41973350904948}
        cy={216.41892036436906}
        fill="#ff0000"
        r={12}
      />
    </svg>
    <svg
      className="label-2"
      height={190}
      viewBox="0 0 190 190"
      width={190}
    >
      <circle
        cx={95}
        cy={95}
        r={83}
        style={
          Object {
            "fill": "#333333",
            "stroke": "#003300",
            "strokeWidth": 24,
          }
        }
      />
      <path
        d="M 95 12 A 83 83, 0, 0, 1, 95.00000000000001,178
              M 95.00000000000001 178 A 83 83, 0, 0, 1, 95.00000000000001,178"
        style={
          Object {
            "fill": "none",
            "stroke": "#00ff00",
            "strokeWidth": 24,
          }
        }
      />
      <circle
        cx={95}
        cy={12}
        fill="#00ff00"
        r={12}
      />
      <circle
        cx={95.00000000000001}
        cy={178}
        fill="#00ff00"
        r={12}
      />
    </svg>
    <svg
      className="label-3"
      height={130}
      viewBox="0 0 130 130"
      width={130}
    >
      <circle
        cx={65}
        cy={65}
        r={53}
        style={
          Object {
            "fill": "#333333",
            "stroke": "#01252d",
            "strokeWidth": 24,
          }
        }
      />
      <path
        d="M 65 12 A 53 53, 0, 0, 1, 65,118
              M 65 118 A 53 53, 0, 0, 1, 14.594004636356857,48.6220992981278"
        style={
          Object {
            "fill": "none",
            "stroke": "#05bae0",
            "strokeWidth": 24,
          }
        }
      />
      <circle
        cx={65}
        cy={12}
        fill="#05bae0"
        r={12}
      />
      <circle
        cx={14.594004636356857}
        cy={48.6220992981278}
        fill="#05bae0"
        r={12}
      />
    </svg>
  </div>
</div>
`;
